portfolio/src/main/java/com/google/sps/data/InternalTags.java

```diff
@@ -24,9 +24,13 @@
   private static void setTags() {
     INTERNAL_TAGS.add(DEFAULT_TAG);
     INTERNAL_TAGS.add("#business");
+    INTERNAL_TAGS.add("#animals");
```

Comment:
If you define them as variables, you could be doing
INTERNAL_TAGS.add(ANIMALS_TAG);